examples/gnrc_networking_mac/Makefile

```diff
@@ -36,9 +36,9 @@ USEMODULE += netstats_l2
 USEMODULE += netstats_ipv6
 USEMODULE += netstats_rpl
 # Use GoMacH as the MAC layer protocol
-USEMODULE += gnrc_gomach
+# USEMODULE += gnrc_gomach
 # In case of using LWMAC MAC protocol instead of GoMacH, uncomment the following line and comment the above line
-# USEMODULE += gnrc_lwmac
+USEMODULE += gnrc_lwmac
```

Comment:
Comment above is not in sync with the Makefile. (It is already uncommented.)

Reply:
Is this a deliberate change, or did changing the MAC protocol in the example just sneak in? To me this seems not to be relevant to this PR.

Reply:
Oops, yes this snuck in :sweat_smile:

Reply:
Fixed